### PR TITLE
Document making a command tenant-aware

### PR DESCRIPTION
**Pull request by lukinovec**
I copied and modified the page from v2 with changes from https://github.com/archtechx/tenancy/pull/869 in mind. I left a todo in the code – this should not get merged before the mentioned PR gets merged (and before pending tenants get documented).

### CODE REVIEW

**Comment by stancl** [comment]
Maybe you can change this to follow v3 code (therefore different trait name and no pending logic), we can merge this, and then in v4 docs we make the necessary adjustments?

**Comment by lukinovec** [comment]
> Maybe you can change this to follow v3 code (therefore different trait name and no pending logic), we can merge this, and then in v4 docs we make the necessary adjustments?

Okay, looks like there are no changes to tenant-aware commands since v2, so I copied the v2 page and made some casing adjustments.

https://github.com/stancl/tenancy-docs/pull/196/commits/84e1992c15f6e4ad5985c7ceb39317508020f5c9

File: source/docs/v3/tenant-aware-commands.blade.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+class MyCommand extends Command
+{
+    use TenantAwareCommand;
+}
+```
+
+However, this trait requires you to implement a `getTenants()` method that returns an array of `Tenant` instances.
+
+If you don't want to implement the options/arguments yourself, you may use one of these two traits:
+- `HasATenantsOption` - accepts multiple tenant IDs, optional -- by default the command is executed for all tenants
```
Comment (stancl) [inline source/docs/v3/tenant-aware-commands.blade.md:23]: Is this the name of the trait in v3?

Comment (lukinovec) [reply source/docs/v3/tenant-aware-commands.blade.md:23]: Just double-checked, and yeah, it is

image

**Comment by lukinovec** [comment]
Can we close https://github.com/stancl/tenancy-docs/issues/63 now that this is merged?

**Pull request merged**